lib/wings/active_fedora_converter.rb

```diff
@@ -114,16 +114,20 @@ class DefaultWork < ActiveFedora::Base
       class_attribute :valkyrie_class
       self.valkyrie_class = Hyrax::Resource
 
-      def self.model_name(*)
-        _hyrax_default_name_class.new(valkyrie_class)
-      end
+      class << self
+        def model_name(*)
+          _hyrax_default_name_class.new(valkyrie_class)
+        end
 
-      def self.human_readable_type
-        valkyrie_class.human_readable_type
-      end
+        def human_readable_type
+          valkyrie_class.human_readable_type
+        end
 
-      def self.to_rdf_representation
-        "Wings(#{valkyrie_class})"
+        def inspect
+          "Wings(#{valkyrie_class})"
+        end
+        alias_method :to_rdf_representation, :inspect
```

Comment:
actually slightly tempted to invert this:

```ruby
def to_rdf_representation
  "Wings(#{valkyrie_class})"
end
alias_method :inspect, :to_rdf_representation
alias_method :to_s, :inspect
```

this would limit the damage if `inspect` were to get overridden elsewhere.